core/src/main/java/org/mapfish/print/map/tiled/wms/TiledWmsLayerParam.java

```diff
@@ -18,6 +18,7 @@
      * A two element array of integers indicating the x and y size of each tile.
      */
     public int[] tileSize;
+    public int[] tileBufferSize;
```

Comment:
Probably @HasDefaultValue is missing ...

Reply:
See: https://github.com/mapfish/mapfish-print/runs/1880797374?check_suite_focus=true

Reply:
Thanks, added.